src/DotVVM.Framework/Resources/Scripts/metadata/primitiveTypes.ts

```diff
@@ -192,3 +194,17 @@ function validateTimeSpan(value: any) {
         return { value: serializeTimeSpan(value), wasCoerced: true };
     }
 }
+
+function validateDateTimeOffset(value: any) {
+    if (typeof value === "string") {
+        // strict DotVVM format parse
+        if (serializationParseDateTimeOffset(value)) {
+            return { value };
+        }        
+    }
+    
+    // TODO: support conversion to date
```

Comment:
I don't think we should ever have support for conversion of Date to DateTimeOffset, since Date does not have the offset information

Reply:
Agree - we have to keep our string-based format in the viewmodel so the offset information is persisted. 
However, if anyone wants to do something with the value on the client, they'll probably need some function that will be able to parse this value - e. g. return Date and offset, or better return our DateTimeOffset representation.